rbac/api/cross_access/view.py

```diff
@@ -82,8 +83,10 @@ class CrossAccountRequestViewSet(
     """
 
     permission_classes = (CrossAccountRequestAccessPermission,)
-    filter_backends = (filters.DjangoFilterBackend,)
+    filter_backends = (filters.DjangoFilterBackend, OrderingFilter)
     filterset_class = CrossAccountRequestFilter
+    ordering_fields = ("request_id", "start_date", "end_date", "created", "modified")
```

Comment:
Can you also allow ordering by first_name and last_name for the approver view?

Reply:
Unfortunately, that's nasty implementation because we only store the user_id on our side, we fetch the first_name and last_name, then we have to sort the result dict by first_name or last_name.

We still could do that but just not a clean way (sorting a dict doesn't feel like a right thing once the number of request gets large). If that's really desired feature, I'd defer that to the future lease.

Reply:
After taking a [closer look at the designs](https://marvelapp.com/prototype/257je526/screen/74764732) it looks like you're right and I'll remove being able to sort those 2 columns.

Reply:
Cool! BTW, I just look at the design and it requires ability to filter by first/last name. This would encounter the same situation: we search for all the users (db query) and replace the user_id with user's info (call BOP and do it in rbac). So if we want to filter by first/last name, we have to do that for the return result (not like other filter that performs in the db query). Therefore, it would be no better than just get all the request result and search for it on the UI side. I would defer that to future release too if that's really desired. We could discuss with Margot about it

Reply:
Okay, I can easily remove those filters. Is filtering by first/last name in the approver view essential @mmenestr?

Reply:
It is pretty essential in that it's the only way to search for _who_ sent you a request! So I would say, please implement it if you can 👍

Reply:
If there's a way to handle this in the UI even as an interim solution, that may be the best. Otherwise as Jay mentions, we'll quickly run into issues with pagination (with potential for timeout from IT). If so, we can/should log something to address this in a future iteration. It's a non-trivial amount of work, and something we have identified as a limitation for some of the other /principals/ endpoints as well.

Reply:
All I can do on the frontend is filter the 10-100 rows that are on the page (and from my understanding, your scaling problem still applies to large page sizes since you'll have to make 100 requests to the BOP for a page size of 100). This will be a confusing filter since it does not behave like the other filters. Can you think of any other ways in the UI to handle not being able to filter by first/last name on the backend @coderbydesign or @mmenestr ?

Reply:
Based on slack conversation, we'll not be implementing this on this iteration.